Integrations/EWSv2/EWSv2.yml

```diff
@@ -77,6 +77,11 @@ configuration:
   name: requestTimeout
   required: false
   type: 0
+- display: Max incidents per fetch
+  name: maxFetch
+  defaultvalue: "50"
+  type: 0
+  required: true
```

Comment:
According to the hooks, this breaks backwards compatibility.
Even though this has a default value here, the existing integration configs in the DB don't have this key.
This should be OK, since you handle that case in the code, but:
1. The hook doesn't check that, so you will need to bypass it somehow...
2. There is no guarantee that an existing configuration will survive the content update (the integration may still fail simply because the server logic requires all mandatory fields to have a value, and the default value is not applied to existing configurations, this needs further testing).

The simpler solution is to set `required: false` (and handle the case where it is empty in the code, which you already do).